apps/wallet/src/ui/app/staking/validators/SelectValidatorCard.tsx

```diff
@@ -65,45 +98,97 @@ export function SelectValidatorCard() {
     }
 
     return (
-        <BottomMenuLayout className="flex flex-col w-full items-center m-0 p-0">
+        <div className="flex flex-col w-full -my-5">
```

Comment:
Curious why we need negative y margin here. Not always an issue but can point to larger layout issues.

Reply:
There is default padding top and bottom of the parent container. This content should be at full height.